client/src/containers/Header/Header.js

```diff
@@ -10,7 +10,7 @@ import MenuIcon from '@material-ui/icons/Menu';
 
```

Comment:
why did you add a comma here?

Reply:
my bad :( I had tried adding style changes there, but then i removed it and missed the comma.